packages/editor-ui/src/composables/usePinnedData.test.ts

```diff
@@ -133,4 +141,71 @@ describe('usePinnedData', () => {
 			expect(spy).toHaveBeenCalled();
 		});
 	});
+
+	describe('canPinData()', () => {
```

Comment:
Thank you for adding Unit tests. Would it also be useful to add some e2e tests as well for this bug?

Reply:
Especially if we now drop the condition in the vue file this unit test should now have us fully covered.

I'm a bit unclear still on when to add e2e tests, I understand we're looking to move away from them "by default" and that there is a real cost associated?

Reply:
there's no rule. My general rule is if the bug depends on the interaction of multiple modules, then better to have a test for it.